### ISSUE TITLE
Playlist support does not work on the `music.youtube.com` subdomain

### ISSUE THREAD

**Issue by watermelonpizza** [deepjyoti30/ytmdl#140]
# Bug report

## Checklist

- [x] I have searched for similar bugs in the issues.
- [x] I will add the bug label to this issue.

## Version Info

- Ytmdl version: `2020.11.20-1`
- Youtube-dl version: `2020.12.31`

## How Ytmdl was installed?
Ubuntu 18.04.2 LTS `pip3 install ytmdl`

## Console Log

```console
$ ytmdl https://www.youtube.com/playlist?list=OLAK5uy_ljerJWCSJQcdG-LsoL_26t-B94DNP5Dog
 ==>  Youtube playlist passed...extracting!
 ==>  Playlist: Album - The Wall (Buunshin Remix) / If U (Synergy Remix)
 ==>  2 songs found
 ==>  Current extracted title for the song is: `the wall buunshin remix`
 ==>  Most extracted titles are not accurate and they affect the meta search
Would you like to change?[Y/n] ^C ==>
Exiting..!
```

```console
$ ytmdl https://music.youtube.com/playlist?list=OLAK5uy_ljerJWCSJQcdG-LsoL_26t-B94DNP5Dog
 ==>  Searching to see if already present in /mnt/c/Users/danielm
 ==>  Searching Youtube for https://music.youtube.com/playlist?list=OLAK5uy_ljerJWCSJQcdG-LsoL_26t-B94DNP5Dog
 ==>  No search results found!
 ==>  No song found. Please try again with a different keyword.
```

## Description

When using `www.youtube.com` with a playlist url the application detects it is a playlist, however when using `music.youtube.com` this doesn't work.
e.g. https://www.youtube.com/playlist?list=OLAK5uy_ljerJWCSJQcdG-LsoL_26t-B94DNP5Dog works fine
however https://music.youtube.com/playlist?list=OLAK5uy_ljerJWCSJQcdG-LsoL_26t-B94DNP5Dog doesn't.

**Comment by deepjyoti30** [comment]
@watermelonpizza Thanks for reporting this. Should be fixed in 4c7b41ce37c8edc85a66e1b60f772a1042b2a78f

It was a simple fix since `youtube-dl` is able to extract playlists from `YoutubeMusic` pretty easily as well. Just had to change a regex to match playlists that had the `Youtube Music` domain in it.

**Comment by deepjyoti30** [comment]
Added in `2021.1.14`.
